src/main/java/org/isf/patient/service/PatientIoOperationRepository.java

```diff
@@ -27,31 +27,25 @@
 import org.isf.patient.model.Patient;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
-import org.springframework.transaction.annotation.Transactional;
 
 @Repository
 public interface PatientIoOperationRepository extends JpaRepository<Patient, Integer>, PatientIoOperationRepositoryCustom {
 
-	List<Patient> findByDeletedOrDeletedIsNull(String deletionStatus);
+	List<Patient> findAll();
+	
+	@Query("select p from Patient p order by p.secondName, p.firstName")
+	List<Patient> findAllOrderByName(Pageable pageable);
 
-	List<Patient> findAllByDeletedIsNullOrDeletedEqualsOrderByName(String patDeleted, Pageable pageable);
+	@Query("select p from Patient p where p.name = :name order by p.secondName, p.firstName")
+	List<Patient> findByNameOrderByName(@Param("name") String name);
 
-	@Query("select p from Patient p where p.name = :name and (p.deleted = :deletedStatus or p.deleted is null) order by p.secondName, p.firstName")
-	List<Patient> findByNameAndDeletedOrderByName(@Param("name") String name, @Param("deletedStatus") String deletedStatus);
+	@Query("select p from Patient p where p.code = :id")
+	List<Patient> findAllWhereId(@Param("id") Integer id);
 
-	@Query("select p from Patient p where p.code = :id and (p.deleted = :deletedStatus or p.deleted is null)")
-	List<Patient> findAllWhereIdAndDeleted(@Param("id") Integer id, @Param("deletedStatus") String deletedStatus);
-
-	@Modifying
-	@Transactional
-	@Query(value = "update Patient p set p.deleted = 'Y' where p.code = :id")
-	int updateDeleted(@Param("id") Integer id);
-
-	List<Patient> findByNameAndDeleted(String name, String deletedStatus);
+	List<Patient> findByName(String name);
```

Comment:
`findAllWhereIdAndDeleted` has been replaced with `findAllWhereId` -> ok
`findByNameAndDeletedOrderByName` has been replaced with `findByName` -> ko, what about the `OrderBy`?

Reply:
ops, sorry, I've seen now `findByNameOrderByName`